backend/internal/storage/storage.go

```diff
@@ -27,6 +28,11 @@ type UserRating interface {
 	GetAllUserRatings(context.Context, uuid.UUID) ([]models.UserRating, error)
 }
 
+type Venues interface {
```

Comment:
I think we should combine Venue and Venue Ratings interfaces. IMO they provide functionality over the same data structure and belong together